barretenberg/cpp/src/barretenberg/dsl/acir_format/mock_verifier_inputs.test.cpp

```diff
@@ -38,9 +39,47 @@ TYPED_TEST(MockVerifierInputsTest, MockOinkProofSize)
 {
```

Comment:
nit: there isn't much point in preserving these tests typed given that their functionality for each flavor is entirely different,  I would rather split them in three tests.